upgrade.sh

```
@@ -2521,7 +2546,33 @@ run_env_refresh() {
   arthexis_timing_end "$timing_label"
 }
 
+run_env_dependency_refresh() {
+  local timing_label="${1:-env_refresh_deps}"
+  local status
+
+  echo "Refreshing dependencies before Control enabled-apps lock..."
+  arthexis_timing_start "$timing_label"
+  if FAILOVER_CREATED=1 ./env-refresh.sh --deps-only $ENV_ARGS; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)